### PR TITLE
ci: Implement OBS packaging CI

### PR DESCRIPTION
**Pull request by jcvenegas**
- OBS packages are build based on kata head
 - The OBS kata branch is created on demand
 - TODO: Delete branch when is not needed anymore

### CODE REVIEW

**Comment by jcvenegas** [comment]
Depends on #603

File: .ci/azure/pipelines/obs-packaging-ci.yml [hunk 1]
```diff
@@ -11,9 +11,47 @@ pool:
...
+
+- bash: |
+   # FIXME: Added just for debug
+   export DEBUG=true
+   set -x
+   # This is set by azure, but we use the same variable
+   export AGENT_VERSION=""
+   # this should exist to download versions.yaml
+   # FIXME
+   mkdir -p /home/vsts/go/src/github.com/kata-containers/runtime/
```
Comment (grahamwhaley) [inline .ci/azure/pipelines/obs-packaging-ci.yml]: Should this be constructed from a combo of `${HOME}` and `${GOPATH}` maybe?

**Comment by grahamwhaley** [comment]
I guess you are testing out and still debug/developing the script, so I'll flag as DNM.
I see we can click the link to the Azure build fail, and after a little fiddling, I found the quite nice log UI to browse through the steps and to see the errors :-)

**Comment by jcvenegas** [comment]
Closing this and opening other, I have some issues getting some env variables I want to verify if that helps

**Pull request closed without merging**